Dockerfile

```diff
@@ -41,6 +52,7 @@ RUN RUSTFLAGS='-L/usr/lib -Ctarget-feature=-crt-static' cargo build --release -p
 #######################################
 FROM python:3.8-alpine AS python-builder
 
+RUN apk -U upgrade --no-cache
```

Comment:
Do we need `-U` (aka `--update-cache`) if we're using `--no-cache`? I think it doesn't make sense.

Reply:
Well, sort of: my understanding is that if you use --no-cache then nothing ever gets written to a cache, and previously cached files are ignored as well.

This would also imply that running `apk update` (or adding `-U`) would make no sense, since even if the index files were downloaded and stored in the cache those would be ignored because of `--no-cache`.

Reply:
Oki after a lot of pondering and testing I understand why the -U is unnecessary, removed it. Thanks :)